ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.13__okd.yaml

```diff
@@ -53,7 +53,7 @@ tests:
   container:
     from: src
 - as: aws-e2e-operator
-  cron: 0 */4 * * *
+  cron: 0 0 * * *
```

Comment:
thoughts on making these only Monday-Friday?
`0 0 * * 1-5`

Reply:
I'd be ok with that

Reply:
I will do this in the next PR.. just going to make sure this version of the template works for now. [Slack thread ](https://redhat-internal.slack.com/archives/CM4ERHBJS/p1676578098136429)